src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -5234,6 +5234,9 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_ExplicitNullableAttribute" xml:space="preserve">
     <value>Explicit application of 'System.Runtime.CompilerServices.NullableAttribute' is not allowed.</value>
   </data>
+  <data name="ERR_NotNullableOperatorNotReferenceType" xml:space="preserve">
+    <value>The ! operator can only be applied to reference types.</value>
```

Comment:
Probably not right, e.g. `Box<string?>` could be banged.

Reply:
Added `SuppressNullableWarning_ValueType2` test.